### DISCUSSION TITLE
Downloading UDM backups as well

### DISCUSSION THREAD

**Discussion by roynasser** [danielfernau/unifi-protect-video-downloader#408] [Q&A]
Hi,  I just fell onto this project, and I'm starting to implement it, as not having the Unifi Protect writing to my NAS but only to the UDM single drive is not an ideal solution (its for my home so I'm a bit limited with how "pro" I wanna go on this)... Nonetheless, to my question: have you considered supporting copying the UDM (or other unifi hardware) backups also to the NAS?  This is especially useful since the cloud backups arent really something you can easily toy with, and they are only for the full OS, not specific apps (like network or protect individually).

(tried searching forum and issues for "backup" and came up empty handed)

**Comment by danielfernau** [comment]
Hey there and thanks for reaching out! Not sure if this fits the scope of this project, specifically, but I'll keep it in mind and either add it as a bonus feature or just build a separate tool to achieve this.

I'm currently looking into re-writing the tool from scratch anyways, so stay tuned!
